tests/Microsoft.Identity.Test.Integration.netfx/HeadlessTests/ClientCredentialsTests.WithRegion.cs

```diff
@@ -94,6 +86,7 @@ public async Task InvalidRegion_GoesToInvalidAuthority_Async()
 
         private void AssertTelemetry(HttpSnifferClientFactory factory, string currentTelemetryHeader, int placement = 0)
         {
+            Console.WriteLine($"Total requests for regional: {factory.RequestsAndResponses.Count}");
```

Comment:
This looks like a leftover from debugging. Or do you need it?

Reply:
yes, removing it